images/libvirt-kubevirt/libvirtd.sh

```diff
@@ -80,6 +80,11 @@ fi
 
 echo "cgroup_controllers = [ ]" >> /etc/libvirt/qemu.conf
 
+# Configure libvirt SASL authentication
+echo "auth_unix_rw=\"sasl\"" >> /etc/libvirt/libvirtd.conf
+echo "mech_list: scram-sha-1" >> /etc/sasl2/libvirt.conf
+cat /etc/sasl-secret/password | saslpasswd2 -p -c -a libvirt $(cat /etc/sasl-secret/username)
+
 if [[ -n "$LIBVIRTD_DISABLE_TCP" ]]; then
```

Comment:
Use of the TCP socket is insecure unless you can enable Kerberos, which we do not want. UNIX socket or TLS socket are the only options and for local connections it is preferred to use UNIX socket for performance reasons.